src/Security/MemberAuthenticator/ChangePasswordHandler.php

```diff
@@ -77,7 +81,8 @@ public function doChangePassword(array $data)
         $member->write();
 
         if ($member->canLogIn()->isValid()) {
-            $member->logIn();
+            Injector::inst()->get(IdentityStore::class)
```

Comment:
wonder if we should inject the IdentityStore into Security class (as we do with the authenticators), then we can do

Security::singleton()->getIdentityStore()? What do you think about that?